libraries/Microsoft.Bot.Connector/Authentication/AuthenticationConfiguration.cs

```diff
@@ -12,6 +12,31 @@ namespace Microsoft.Bot.Connector.Authentication
     /// </remarks>
     public class AuthenticationConfiguration
     {
-        public string[] RequiredEndorsements { get; set; } = new string[] { };
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuthenticationConfiguration"/> class.
+        /// </summary>
+        public AuthenticationConfiguration()
+            : this(null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuthenticationConfiguration"/> class.
+        /// </summary>
+        /// <param name="claimsValidator">A <see cref="ClaimsValidator"/> instance used to validate claims.</param>
+        public AuthenticationConfiguration(ClaimsValidator claimsValidator)
```

Comment:
Why have a constructor with a specific property out of both properties? When we add more properties will we add more ctors? Why not leave this as a data transfer object, i.e. default ctor + properties?

Reply:
Having in the constructor simplifies DI, let's chat offline about the change, we can explore other options to pass this in.

Reply:
Removed constructors and added setter.